src/styles/index.scss

```diff
@@ -1,3 +1,115 @@
-body {
+@import '//fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;1,100&display=swap';
+
+* {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
+}
+
+.card {
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+
+  &__img {
+    height: 134px;
+    width: 160px;
+    margin-inline: 3px;
+    background-image: url(../images/imac.jpeg);
+    background-position: top;
+    background-repeat: no-repeat;
+    background-size: contain;
+  }
+
+  &__text {
+    &--head {
+      margin-top: 40px;
+      font-size: 12px;
+      font-weight: 500;
+      line-height: 15px;
+    }
+
+    &--grey {
+      margin-top: 4px;
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 12px;
+      color: #616070;
+    }
+  }
+
+  &__rating {
+    margin-top: 16px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: flex-end;
+
+    &-text {
+      font-size: 10px;
+      line-height: 12px;
+      font-weight: 400;
+    }
+  }
+
+  &__price {
+    margin-top: 24px;
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+
+    &-text {
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 15px;
+      color: #616070;
+    }
+
+    &-number {
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 20px;
+    }
+  }
+
+  &__button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-decoration: none;
+    margin-top: 16px;
+    width: 100%;
+    background-color: #00acdc;
+    border-radius: 5px;
+    color: #fff;
+    font-size: 14px;
+    font-weight: 700;
+    height: 40px;
+    text-transform: uppercase;
+    border: 1px solid #00acdc;
+
+    &:hover {
+      background-color: #fff;
+      color: #00acdc;
+    }
+  }
+}
+
+.stars {
+  display: flex;
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    background-position: center;
+    background-image: url(../images/star-active.svg);
+    background-repeat: no-repeat;
+    margin-right: 4px;
+
+    &--not_active {
```

Comment:
just use styles from the previous task - there should be styles for each amount of active stars (--0, --5):

![image](https://github.com/user-attachments/assets/dc1e6f4b-b0a9-417c-93e1-ed647f48b9d4)